src/app.ts

```diff
@@ -98,16 +99,14 @@ export async function initSlack(): Promise<void> {
 }
 
 export async function initDiscord(): Promise<void> {
+  const discordBotToken = await Config.findOne('discordBotToken');
   try {
-    if (env.discordBotToken) {
+    if (discordBotToken?.value) {
```

Comment:
With the change above, this will be the value directly now and will have a type of `string | null`

```suggestion
    if (discordBotToken) {
```